src/components/SearchQueryFetchConnector/SearchQueryFetchConnector.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+// @ts-ignore
+import SolrConnector from 'react-solr-connector';
+import SolrConnectorDemo from './SolrQueryFetchConnector';
+
+class SolrQueryFetch extends React.Component {
```

Comment:
This should be exported, e.g.

`export class SolrQueryFetch extends React.component {`

It means we can then either do a named or default import later:

named import:

`import { SolrQueryFetch } from 'x.js'`

default import:

`import SolrQueryFetch from 'x.js'`

(both do the same thing)